openwisp_users/tests/test_admin.py

```diff
@@ -692,6 +694,53 @@ def test_organization_owner(self):
         )
         self.assertContains(response, 'tester')
 
+    def test_first_orguser_creates_org_owner(self):
+        org = self._get_org()
+        user = self._get_user()
+        org_user = self._create_org_user(organization=org, user=user, is_admin=True)
+        org_owner_qs = OrganizationOwner.objects.all()
+        self.assertEqual(org_owner_qs.count(), 1)
+        org_owner = org_owner_qs.first()
+        self.assertEqual(org_owner.organization, org)
+        self.assertEqual(org_owner.organization_user, org_user)
+
+    def test_first_orguser_creates_no_org_owner(self):
+        org = self._get_org()
+        user = self._get_user()
+        self._create_org_user(organization=org, user=user, is_admin=False)
+        org_owner_qs = OrganizationOwner.objects.all()
+        self.assertEqual(org_owner_qs.count(), 0)
+
+    def test_second_orguser_creates_no_org_owner(self):
+        org = self._get_org()
+        user = self._get_user()
+        org_user = self._create_org_user(organization=org, user=user, is_admin=True)
+        user1 = self._create_user(username='user1', email='user1@gmail.com')
+        self._create_org_user(organization=org, user=user1, is_admin=True)
+        org_owner_qs = OrganizationOwner.objects.all()
+        self.assertEqual(org_owner_qs.count(), 1)
+        org_owner = org_owner_qs.first()
+        self.assertEqual(org_owner.organization, org)
+        self.assertEqual(org_owner.organization_user, org_user)
+
+    def test_view_org_owner_org_admin(self):
+        self.client.force_login(self._get_admin())
+        r = self.client.get(reverse(f'admin:{self.app_label}_organization_add'))
+        self.assertEqual(r.status_code, 200)
+        self.assertContains(r, 'Organization owners')
+
+    @patch.object(
+        OrganizationOwner, 'full_clean', side_effect=ValidationError('validation error')
+    )
+    def test_invalid_org_owner(self, mocked):
```

Comment:
patch the exception method of logger and assert it has been called, see how it's done here:
https://github.com/openwisp/openwisp-network-topology/pull/73/files#diff-13c0cc68f2519b129b31a58d282842d4R112-R115